tests/test_unit.py

```diff
@@ -44,6 +44,17 @@ def test_copy():
     assert slug == ureg.slug
 
 
+def test_compatibility():
+    ureg = ansunits.UnitRegistry()
+    length_units = {"cm", "in", "m"}
+    force_units = {"dyne", "lbf", "pdl"}
+    temperature_difference_units = {"delta_C", "delta_R", "delta_F"}
+
+    assert ureg.ft.compatible_units() == ft
```

Comment:
```suggestion
    assert ureg.ft.compatible_units() == length_units
```
:-)  The tests are failing